### PR TITLE
Red race route: fix a lot of typos/formatting/errors

### PR DESCRIPTION
**Pull request by Grogir**
Besides typos and formatting some menuing steps were missing, it now correctly reflects the actual route used in the 2020 finals

### CODE REVIEW

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # Pokemon Red Glitchless Race Route (NO IT-FlexLass-3Repels-PPUp)
 
-**This is one of the many race routes people use and most of them are as good. I decided to document this because it was used by both 3 finalist at the 2020 Red Tournament**
-**You can find the VOD of the Finals[here](https://youtu.be/owoeSjzdqL0), it's a good example of this route in action**
+**This is one of the many race routes people use and most of them are as good. I decided to document this because it was used by all 3 finalists at the 2020 Red Tournament.**
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/main-glitchless/race-route/README.md:3]: extra space at the end here

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md [hunk 2]
```diff
@@ -45,42 +45,41 @@
...
+	- Swap to Squirtle, then Bubble spam
 * Onix [Heal if HP < 7]
-	- Swap to squirtle
-		* If Onix use bide on switch in: Tail Whip x2
-    * If Onix use bide on the turn you bubble:
-      - Use bubble again with good special or if you get a crit
-      - Use bubble again, then switch to pidgey with bad Special
-	- Bubble Spam
+	- Swap to Squirtle
+		- If Onix uses bide on switch in: Tail Whip x2
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/main-glitchless/race-route/README.md]: if you're going for capitals on move names, Bide here

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md [hunk 3]
```diff
@@ -169,19 +168,19 @@
...
 
 - Use the Center
-- Pickup the Hidden rare candy [here](http://gunnermaniac.com/pokeworld?map=1#235/44)
+- Pick up the hidden Rare Candy [here](http://gunnermaniac.com/pokeworld?map=1#235/44)
 
 **Rival:**
 * Pidgeotto: HA x3 (If you get some bad rolls use MP to kill)
-	- If you get hit by 2 sand attacks swap to Squirtle and let it die
-      - Try to put Pidgeotto in HA range before doing this: doing so avoids getting a chance to get sand-attack again after swapping back
+  - If you get hit by 2 sand attacks swap to Squirtle and let it die
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/main-glitchless/race-route/README.md]: Sand Attack

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md [hunk 4]
```diff
@@ -275,28 +277,28 @@ If you still miss a cutter:
...
+- Teach TM 11 (BB) in slot 4
 
 **Rival:**
 - Pidgeotto: HA x2
 	* If you get sand attacked swap to another pokemon and let it die
 * Rat: MP
 * Kadabra: Thrash
 
-If you have less than 9 HP, Fight the Gentleman in [this room here](https://gunnermaniac.com/pokeworld?map=96#21/11) with thrash then pickup the rare candy:
+If you have less than 9 HP, Fight the Gentleman in [this room here](https://gunnermaniac.com/pokeworld?map=96#21/11) with Thrash then pick up the Rare Candy
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/main-glitchless/race-route/README.md]: Fight->fight

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md [hunk 5]
```diff
@@ -353,21 +356,22 @@ If you have less than 9 HP, Fight the Gentleman in [this room here](https://gunn
...
 **Gambler:**
 * Growlithe:
 	- If you have less than 5 HP Potion
 	- BB
 * Vulpix: Thrash
 
 ## Underground
-- Get the Hidden Elixer [here](http://gunnermaniac.com/pokeworld?map=121#21/5)
-	- Get the Hidden Nugget [here](http://gunnermaniac.com/pokeworld?map=121#12/2)
+- Get the hidden Elixer [here](http://gunnermaniac.com/pokeworld?map=121#21/5)
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/main-glitchless/race-route/README.md]: perhaps link on the words [hidden Elixer] and drop "here" ? (same for next line)

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md [hunk 6]
```diff
@@ -471,19 +474,19 @@ NOTE: From this point on you have 2 revives which means deaths aren't as scary,
 ## Cycling Road
 - NOTE: You can hold B to stop automatically moving down on Cycling Road:
 - Get the Rare Candy found [here](http://gunnermaniac.com/pokeworld?map=1#125/148)
-- Get the hidden PPUp found [here](https://gunnermaniac.com/pokeworld?map=1#127/206)
+- Get the hidden PP Up found [here](https://gunnermaniac.com/pokeworld?map=1#127/206)
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/main-glitchless/race-route/README.md]: same idea linking on "hidden PP Up" and dropping "here"

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md [hunk 7]
```diff
@@ -536,20 +539,20 @@ NOTE: From this point on you have 2 revives which means deaths aren't as scary,
...
 - EQ x4
-- Walk back to the Teleporter and dig out and fly to
+- Walk back to the Teleporter and Dig out and Fly to
 
 ## Viridian City
 **Cooltrainer:**
 - EQ
 
-SAVE
+SAVE
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/main-glitchless/race-route/README.md:549]: is this change needed?

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md [hunk 8]
```diff
@@ -471,19 +474,19 @@ NOTE: From this point on you have 2 revives which means deaths aren't as scary,
...
 
 ## Fuschia City
 * MENU:
   * Super Repel
-  * Swap slot 8 with X Spec
+  * Swap slot 8 (Potion) with X Speed
   * Teach TM 26 (Earthquake) over Thrash
   * Use Bike
 - In the Safari Zone:
   - If you have 0 potions get the full restore [here](https://gunnermaniac.com/pokeworld?map=217#21/10)
```
Comment (TiKevin83) [inline docs/gen-1/red-blue/main-glitchless/race-route/README.md]: Full Restore could be capitalized here

**Review by TiKevin83 — CHANGES_REQUESTED** [review]
Thanks for your contribution Grogir! Just left a few more small notes for cleanup and some lines with extra space at the end.

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md (hunk 1 above)
Comment (Grogir) [reply docs/gen-1/red-blue/main-glitchless/race-route/README.md:3]: Spaces at the end of a line in markdown are used to make a new line in the same paragraph and i believe thats what was intended here

File: docs/gen-1/red-blue/main-glitchless/race-route/README.md (hunk 7 above)
Comment (Grogir) [reply docs/gen-1/red-blue/main-glitchless/race-route/README.md:549]: Same as above (current doc misses a new line)

**Comment by Grogir** [comment]
I also thought of removing some long explanations that look copied from the beginners route, to lighten a bit the text so its more readable like advanced route. Thoughts?
